packages/ui/rollup.config.js

```diff
@@ -0,0 +1,38 @@
+// rollup.config.js
+import { defineConfig } from 'rollup';
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+
+const config = defineConfig([
+  // CJS config
+  {
+    input: ['src/index.ts'],
+    output: {
+      dir: 'dist',
+      format: 'cjs',
+      sourcemap: true,
+    },
+    external: (id) => /style-dictionary/.test(id),
+    plugins: [typescript({ declarationDir: 'dist/types' }), terser()],
+  },
+  // ESM config
+  {
+    input: ['src/index.ts'],
+    output: {
+      dir: 'dist/esm',
+      format: 'es',
+      preserveModules: true,
+      preserveModulesRoot: 'src',
+      sourcemap: true,
```

Comment:
So will sourcemaps be bundled in for production?

Reply:
Yes, sourcemaps will be bundled for prod per our discussion and see if anyone complains about that.

Reply:
How much impact does that have on bundle size?

Reply:
> Small Nit:
> 
> https://github.com/aws-amplify/amplify-ui/blob/c070cc8a155a351beaf2d5932d85d965ad8e5d26/package.json#L80
> 
> should be removed 😛

That's a good catch

Reply:
Almost 30% increase I think.